modules/nw-multus-delete-network.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-delete-network_{context}"]
+= Removing an additional network attachment definition
+
+As a cluster administrator, you can remove an additional network from your
+{product-title} cluster. The additional network is not removed from any Pods it
```

Comment:
Wait. So am I really removing the network if all of my Pods are still attached to it? Does this match what users would expect when they "remove a network?"

It seems like checking to see which Pods are attached to the network and removing the Pods from the network need to be part of this process. (Which should be in its own assembly.)

Reply:
@dougbtv, to fully remove an additional network would a user need to remove the attachment annotation from every Pod that that network is associated with?

Reply:
Queried on Slack.